packages/Card/src/components/Avatar/Avatar.js

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { ShirtSizes } from "@paprika/helpers/lib/customPropTypes";
+import * as sc from "./Avatar.styles";
+
+const propTypes = {
+  /** Avatar content. It can be initial as a string or icon */
+  children: PropTypes.node,
+  /** Background color of the Avatar */
+  color: PropTypes.oneOf(["black", "blue", "green", "orange", "pink", "red"]),
+  /** Randomize background color */
+  isRandomBackground: PropTypes.bool,
```

Comment:
this shouldn't be an option, in an advanced scenario the colour is created depending on the name of the user, so if you have this kind of avatar in a chat you have less probability to have two users with the same initial and same background colour.